packages/portfolio-contract/src/portfolio.contract.ts

```diff
@@ -48,6 +65,7 @@ export const contract = async (
   );
 
   const proposalShapes = makeProposalShapes(brands.USDC);
+  const offerArgsShapes = makeOfferArgsShapes();
```

Comment:
if no runtime args are needed to make the shapes, they should be constants, not built by a function.

But I would expect they should contain brands, whose identities can't be known until runtime.